.github/pull_request_template.md

```diff
@@ -1,8 +1,15 @@
 <!-- 
-  - Use [x] to complete the items
-  - Remove the items unrelated to your work
-  - Add any relevant information you consider useful
-  - If there are no reviewers for your language, please mention it explicitly
+Before opening the pull request:
+  
+- Use [x] to complete the items
+- Remove the items unrelated to your work
+- Add any relevant information you consider useful
+- If there are no reviewers for your language, please mention it explicitly
+
+During the review process:
+
+- Please apply suggestions in batches from the "files changed" tab, as explained in https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews.
```

Comment:
to be honest I don't think many will copy and paste the link as it is not clickable in a text field... not sure what to do frankly, other than remove it

Reply:
a) pretty much no one opens pull requests on a phone (for the first time, mind you), which would make it a slight inconvenience
b) what good would removing it do?

Reply:
this is might be unrelated to your concern but maybe this is an improvement in terms of communication?

```suggestion
- Please apply suggestions in batches from the "files changed" tab (see https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews for a visual demo)
```